File: HorseRacing.Contracts/Models/Game/Requests/GetGameRequest.cs
using HorseRacing.Contracts.Base.Dto;
using HorseRacing.Contracts.Base.Requests;

namespace HorseRacing.Contracts.Models.Game.Requests
{
    public class GetGameRequest : BaseRequest<BaseModelDto>
    {
        public GetGameRequest() : base() { }
        public GetGameRequest(BaseModelDto data) : base(data) { }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/requests/get-game/get-game-request.ts
import type { BaseModelDto } from '~/interfaces/api/contracts/base/dto/base-model-dto';
import type { BaseRequest } from '../../../../base/requests/base-request';

export class GetGameRequest implements BaseRequest<BaseModelDto> {
    Data: BaseModelDto;

    constructor(data?: BaseModelDto) {
            this.Data = data || { Id: ''};
        }
}
